### PR TITLE
Update NXDNHost.txt

### PR DESCRIPTION
**Pull request by IU5HJU**
Reflector in numerical order

### CODE REVIEW

File: NXDNGateway/NXDNHosts.txt [hunk 1]
```diff
@@ -12,24 +15,33 @@
...
+
+# 2249 IT SICILIA
+2249 nxdn.digitalsicilia.it 41400
+
 # RU DMR TG2503
 2503 nxdn.r1ik.ru 41400
 
+# 3023 Ontario Crosslink
+3023    ontxlink.hopto.org 41400
+
```
Comment (kp4oo) [inline NXDNGateway/NXDNHosts.txt]: #38 PRadise 3300
3300    pradise3300.from-pr.com    41400

File: NXDNGateway/NXDNHosts.txt [hunk 2]
```diff
@@ -12,24 +15,33 @@
...
+2140  94.177.235.81 41400
+
+# 2225  IT Tuscany
+2225  80.211.99.134 41400
+
+# 2249 IT SICILIA
+2249 nxdn.digitalsicilia.it 41400
+
 # RU DMR TG2503
 2503 nxdn.r1ik.ru 41400
```
Comment (kp4oo) [inline NXDNGateway/NXDNHosts.txt]: #35 Old Melbourne,FL Repeater Group
2700  2700nxdn.ddns.net  41400

File: NXDNGateway/NXDNHosts.txt (hunk 1 above)
Comment (g4klx) [reply NXDNGateway/NXDNHosts.txt]: @kp4oo if you want to add more entries to the file, please submit your own pull request, don't hijack someones elses.

File: NXDNGateway/NXDNHosts.txt (hunk 2 above)
Comment (g4klx) [reply NXDNGateway/NXDNHosts.txt]: @kp4oo if you want to add more entries to the file, please submit your own pull request, don't hijack someones elses.

**Pull request merged**